R/ggplot2.r

```diff
@@ -3,5 +3,5 @@
 
 #' @import scales grid gtable
 #' @importFrom stats setNames
-#' @importFrom rlang quo quos
+#' @importFrom rlang quo quos as_function
```

Comment:
As you use `rlang::`, you don't need to import this.

Reply:
Won't we need it somewhere to avoid CRAN warnings about not being imported? Otherwise, I'm fine removing of course.

Reply:
I don't think so. I suggest removing. If it's a problem the Travis build will break.

Reply:
What do you mean? As long as fully qualified form (`rlang::as_function()`), you don't need to import.